<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_4</class>
 <widget class="QDialog" name="Dialog_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="initial_VLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="choice1_PBtn">
         <property name="text">
          <string>I Love It</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="choice2_PBtn">
         <property name="text">
          <string>Yes I Want</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="choice3_PBtn">
         <property name="text">
          <string>No I Don't Want</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="choice4_PBtn">
         <property name="text">
          <string>WTF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
